src/app/state/rest/permissions.ts

```diff
@@ -13,3 +13,9 @@ export const usePermissions = (options?: Options) => {
       isProtected: true
     })
   }
+
+  export const useHasPermission = (permissionName?: string) => {
+    const [permissions, { isBusy }] = usePermissions()
+    const hasPermission = permissionName && permissions?.includes(permissionName)
```

Comment:
Think this should check for `undefined` otherwise `hasPermission` could become a string. E.g. `permissionName !== undefined && permissions?.includes(permissionName)` or `permissions?.includes(permissionName) ?? false`